prompttools/utils/similarity.py

```diff
@@ -11,8 +11,15 @@
 from typing import Dict
 import pandas.core.series
 import logging
+import warnings
 
-EMBEDDING_MODEL = []  #
+try:
+    import cv2
+except ImportError:
+    cv2 = None
+from skimage.metrics import structural_similarity
```

Comment:
We will have to guard `skimage` as well. Otherwise, it has to be part of our library's requirement